package.json

```diff
@@ -16,51 +16,51 @@
     "lint": "eslint home/javascript/react/** --max-warnings 0"
   },
   "dependencies": {
-    "@babel/cli": "^7.15.7",
-    "@babel/core": "^7.15.8",
+    "@babel/cli": "^7.19.3",
+    "@babel/core": "^7.20.5",
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-    "@babel/preset-env": "^7.15.8",
-    "@babel/preset-react": "^7.14.5",
-    "@fortawesome/fontawesome-free": "^5.15.4",
+    "@babel/preset-env": "^7.20.2",
+    "@babel/preset-react": "^7.18.6",
+    "@fortawesome/fontawesome-free": "^6.2.1",
     "@geneontology/wc-ribbon-strips": "0.0.37",
     "@geneontology/wc-ribbon-table": "0.0.57",
-    "acorn": "^8.5.0",
+    "acorn": "^8.8.1",
     "angular": "1.8.3",
-    "babel-loader": "^8.2.2",
-    "bootstrap": "^4.6.0",
-    "bootstrap-datepicker": "^1.8.0",
+    "babel-loader": "^9.1.0",
+    "bootstrap": "^4.6.2",
+    "bootstrap-datepicker": "^1.9.0",
     "corejs-typeahead": "1.3.1",
     "css-loader": "^3.6.0",
-    "css-minimizer-webpack-plugin": "^2.0.0",
+    "css-minimizer-webpack-plugin": "^4.2.2",
     "exports-loader": "^0.7.0",
     "fast-deep-equal": "3.1.3",
     "file-loader": "^6.2.0",
-    "imagesloaded": "^4.1.4",
-    "immer": "^9.0.7",
-    "jquery-bridget": "2.0.1",
-    "jquery-modal": "0.5.9",
+    "imagesloaded": "^5.0.0",
+    "immer": "^9.0.16",
+    "jquery-bridget": "3.0.1",
+    "jquery-modal": "0.9.2",
     "jquery-stickytabs": "1.2.4",
-    "jquery-validation": "^1.19.3",
+    "jquery-validation": "^1.19.5",
     "jquery.tipsy": "1.0.3",
     "masonry-layout": "^4.2.2",
-    "mini-css-extract-plugin": "^1.6.2",
-    "node-sass": "^5.0.0",
+    "mini-css-extract-plugin": "^2.7.2",
+    "node-sass": "^6.0.1",
```

Comment:
Is it too early to switch to "sass"?

Reply:
I'd rather separate out the switch so any problems from it aren't mixed in with these upgrades.